lib/project_types/script/layers/application/project_dependencies.rb

```diff
@@ -9,23 +9,24 @@ def self.bootstrap(ctx:, language:, extension_point:, script_name:)
 
         def self.install(ctx:, language:, extension_point:, script_name:)
           dep_manager = Infrastructure::DependencyManager.for(ctx, language, extension_point, script_name)
-
           unless CLI::UI::Frame.open(ctx.message('script.project_deps.checking_with_npm')) do
-            return ctx.puts(ctx.message('script.project_deps.none_required')) if dep_manager.installed?
-            begin
-              UI::StrictSpinner.spin(ctx.message('script.project_deps.installing')) do |spinner|
-                dep_manager.install
-                spinner.update_title(ctx.message('script.project_deps.installed'))
-              end
-              true
-            rescue Infrastructure::Errors::DependencyInstallError => e
-              CLI::UI::Frame.with_frame_color_override(:red) do
-                ctx.puts("\n#{e.message}")
-              end
-              false
-            end
-          end
-            raise Infrastructure::Errors::DependencyInstallError
+                   begin
+                     if dep_manager.installed?
+                       ctx.puts(ctx.message('script.project_deps.none_required'))
+                     else
+                       UI::StrictSpinner.spin(ctx.message('script.project_deps.installing')) do |spinner|
+                         dep_manager.install
+                         spinner.update_title(ctx.message('script.project_deps.installed'))
+                       end
+                     end
+                     true
+                   rescue Infrastructure::Errors::DependencyInstallError => e
+                     CLI::UI::Frame.with_frame_color_override(:red) do
+                       ctx.puts("\n#{e.message}")
+                     end
+                     raise e
```

Comment:
Should we update the unit test [here](https://github.com/Shopify/shopify-app-cli/blob/f1aef97051cd6356bda0adf825a868b4397b54b9/test/project_types/script/layers/application/project_dependencies_test.rb#L74) with `assert_raises`?